storage/remote/client.go

```diff
@@ -117,7 +117,7 @@ func (c *Client) Store(ctx context.Context, req []byte) error {
 
 // Name identifies the client.
 func (c Client) Name() string {
-	return fmt.Sprintf("%d:%s", c.index, c.url)
+	return fmt.Sprintf("%s:%s", c.name, c.url)
```

Comment:
Should this be split into two labels now, one `queue`, and one `url`?

Reply:
Yes I think with the addition of the queue name it makes sense for the current label to be split into two. `name` makes more sense as the label name I think, or maybe `rw_name`? Something that would match with the config option.

How much do we care about breaking people's dashboards that use the current label? At the very least the mixin would need to be updated as well.